### DISCUSSION TITLE
We've got something special planned for you..... 🎉 Stay tuned for more updates!

### DISCUSSION THREAD

**Discussion by aimansharief** [Sunbird-Ed/Community#405] [General Discussion]
![SunbirdED (1)](https://user-images.githubusercontent.com/110388201/219407328-43f4c1c5-6b56-4d5a-a578-5a3aaa29586f.png)

@vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @kiranharidas187
@sanyapawah @aayushkedia95 @ChakshuGautam @abhay-samagra
@pkranga @mathewjpallan @mahesh-maney
@pritamps @karn-nahata
@coolbung @parthlawate
@kameshbhr @mmanimegalai @anandvarada
@SanthoshVasabhaktula @vrayulu
@ashokreddy1208 @gandham-santhosh @rajeevsathish @vinukumar-vs @Krishnaj20 @swayangjit
@pallakartheekreddy @rhwarrier @anandp504 @reshmi-nair @surendrasinghs @mmanimegalai @maheshkumargangula
@madhucr @alok-os @pramodkvarma @sukhpreetsamagra @surendrasinghs @pkranga @ddevadat @ravishtrigyn @swayangjit @mathewjpallan @mahesh-maney

**Comment by aimansharief** [comment]
The Sunbird community is  excited to announce the first edition of the SunbirdED Developer BootCamp of 2023 on 2nd & 3rd March at Bangalore.   

Over the course of the bootcamp, we have lined up electrifying sessions for the participants to interact with the SunbirdED community folks, learn by listening to the best practices from the community and hands-on sessions. 

This BootCamp has two tracks:

- SunbirdED DevOps Track - talks from Sunbird adopters and community sharing their best practices and hands-on sessions for the participants to master SunbirdED DevOps automation and installation.

- Sunbird Development Track - series of hands-on sessions covering a wide range of topics for the participants to understand the SunbirdED contribution process.

Architects, technical leads and senior developers  would best benefit from this bootcamp.

Here is the link to the registration form: https://forms.gle/FZYaPRZD891JSs366 to register your participants.
Stay tuned for more updates.


PS: This session is being conducted by the community  and there is no charge but we will be restricting the number of participants for logistic reasons.  So please register early.

**Comment by chitranshu-keshav** [comment]
**Schedule:**

![image](https://user-images.githubusercontent.com/47350594/220336741-2ae68875-f34d-410f-a273-44d7feac6193.png)

**Comment by chitranshu-keshav** [comment]
It is important to make sure that the necessary prerequisites are in place before the bootcamp in order to be productive and get the maximum benefit. Below is the consolidated list of all the necessary Hardware, Software, Skills and the Pre-Learnings required for the same

**Pre-Learning**
_For meaningful participation, Participants are required to go through the following reading materials as well as the videos before the bootcamp_
**Track 1: DevOps**
[Sunbird Ed Installation guide](https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0)
[Tech Deep Dive Session](https://www.youtube.com/watch?v=LcnrvUloanY&t=9s)
**Track 2: Development**
[Sunbird ED Building Blocks](https://sunbird.org/building-blocks)
[Sunbird ED Architecture Overview ](https://ed.sunbird.org/learn/technical-architecture/technical-architecture-diagram)
"Videos:
[Sunbird Overview and Architecture](https://www.youtube.com/watch?v=Uc2aB7eE0Sk&t=7s)
[Tech Deep Dive Session - Sunbird Architecture and Infra Overview](https://www.youtube.com/watch?v=LcnrvUloanY&t=9s)
[Understanding Content Architecture of Sunbird](https://www.youtube.com/watch?v=WxZXaTnj2D0)

**Participant Skill Sets** 
_Event will be most beneficial for Architects, Senior Developers and Tech Leads. Some of the important skill that will be essential to maximize the benefits are_
_**Track 1: DevOps**_
Hands-on experience in administering and debugging Linux systems
Hands-on experience using Docker and Kubernetes to run containerized services
Experience in Ansible, Jenkins and Helm
Hands on experience with any one public cloud service providers e.g. AWS, Azure, GCP, OCI
_**Track 2: Development**_
This track requires the participants to have following technical skills
Angular, NodeJS, JAVA, Akka, Scala, Redis, Cassandra


**Hardware Pre-requisites** 
_Participant are expected to bring their own laptops for the event with below configuration_
**_Track 1: DevOps_** 
Laptop(>2 core, >8GB RAM) with SSH client configured
_**Track 2: Development**_
Laptop (>2 core, >8GB RAM)

**Software Pre-Requisites**
Participants should have the following softwares pre installed in their laptops based on their respective tracks
![image](https://user-images.githubusercontent.com/47350594/221189529-d8785a08-2c24-42d8-ad2e-489d1e803573.png)
Git ->[ Link](https://github.com/git-guides/install-git)
SSH Client -> [Link](https://www.ssh.com/academy/ssh/putty/windows/install)
Angular -> [Link](https://angular.io/cli)
Scala -> [Link](https://www.scala-lang.org/download/)

**Comment by PoojaJManjooran** [comment]
### Fill up the form | Gear-up for the SunbirdED Bootcamp!
 
Kindly mention your GitHub id and the track that you want to choose in the [Form](https://forms.gle/rmbyNQoeKXbGsCzX7) by today, 06:00 PM

**Comment by AmiableAnil** [comment]
Sunbird-Lern developer setup guide and problem statements for the bootcamp.
[Sunbird ED Developer Bootcamp 2023.pdf](https://github.com/Sunbird-Ed/Community/files/10868754/Sunbird.ED.Developer.Bootcamp.2023.pdf)

Or use this [link](https://docs.google.com/document/d/180FWMkmmk0Mh7zycyhbxAxAjCSweG3XGo94aF59XJeU/edit?usp=sharing)

**Comment by pallakartheekreddy** [comment]
Sunbird InQuiry installation guide for the bootcamp

[Sunbird ED Developer Bootcamp 2023 - Sunbird inQuiry.pdf](https://github.com/Sunbird-Ed/Community/files/10868779/Sunbird.ED.Developer.Bootcamp.2023.-.Sunbird.inQuiry.pdf)
